### DISCUSSION TITLE
QSO for SWL function

### DISCUSSION THREAD

**Discussion by Nimesis89** [danleetw/FT8TW#3] [Q&A] [answered]
Function "Save QSO for SWL" is not working.
Regardless of the setting, the log remains empty. The receiver worked for several hours, but one log entry was not saved.

**Comment by danleetw** [comment, marked as answer]
> Function "Save QSO for SWL" is not working. Regardless of the setting, the log remains empty. The receiver worked for several hours, but one log entry was not saved.

The SWL log function should be in a web-based GUI. Go to the Log function, hit the Download button, and then click on the URL.

**Comment by danleetw** [comment]
I found the issue you mentioned. I’ll upload the fix to GitHub shortly.
